pkg/util/labels/labels.go

```diff
@@ -1,4 +1,4 @@
-package kubeobjects
```

Comment:
IMO, this should be under `kubeobjects`, as we only add labels to kubernetes object

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/d7ce7d507fe13a29517cf88b4a4f5234f12425e0